deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -623,29 +623,67 @@ def set_module_attribute(self, module_name, attribute, value):
         self.modules[module_name][attribute] = value
 
 
-class Inventory:
-    def __init__(self, os, inventory_sample=None):
-        self.os = os
+class GeneratorSyscollector:
+
+    def __init__(self, agent_name, batch_size):
+        self.current_batch_events = -1
+        self.current_batch_events_size = 0
+        self.list_events = ['network', 'port', 'hotfix',
+                            'process', 'packages', 'OS', 'hardware']
+        self.agent_name = agent_name
+        self.batch_size = batch_size
         self.SYSCOLLECTOR = 'syscollector'
         self.SYSCOLLECTOR_MQ = 'd'
-        self.inventory = []
-        self.inventory_path = ""
-        self.inventory_sample = inventory_sample
-        self.setup()
 
-    def setup(self):
-        if self.inventory_sample is None:
-            inventory_files = os.listdir(f"inventory/{self.os}")
-            self.inventory_path = f"inventory/{self.os}/{choice(inventory_files)}"
+    def format_event(self, message_type):
+        message = syscollector.SYSCOLLECTOR_HEADER
+        if message_type == 'network':
+            message += syscollector.SYSCOLLECTOR_NETWORK_EVENT_TEMPLATE
+        elif message_type == 'process':
+            message += syscollector.SYSCOLLECTOR_PROCESS_EVENT_TEMPLATE
+        elif message_type == 'port':
+            message += syscollector.SYSCOLLECTOR_PORT_EVENT_TEMPLATE
+        elif message_type == 'packages':
+            message += syscollector.SYSCOLLECTOR_PACKAGES_EVENT_TEMPLATE
+        elif message_type == 'os':
+            message += syscollector.SYSCOLLECTOR_OS_EVENT_TEMPLATE
+        elif message_type == 'hardware':
+            message += syscollector.SYSCOLLECTOR_HARDWARE_EVENT_TEMPLATE
+        elif message_type == 'hotfix':
+            message += syscollector.SYSCOLLECTOR_HOTFIX_EVENT_TEMPLATE
+        elif 'end' in message_type:
+            message += '}'
+
+        today = date.today()
+        timestamp = today.strftime("%Y/%m/%d %H:%M:%S")
+
+        event_map = [
+                        ('<agent_name>', self.agent_name), ('<random_int>', str(randint(1, 10 * 10))),
+                        ('<random_string>', ''.join(SystemRandom().choice(string.ascii_uppercase + string.digits)
+                                            for _ in range(10))),
+                        ('<timestamp>', timestamp), ('<syscollector_type>', message_type)
+                    ]
+
+        for variable, value in event_map:
+            message = message.replace(variable, value)
+
+        message = "{0}:{1}:{2}".format(self.SYSCOLLECTOR_MQ, self.SYSCOLLECTOR, message)
+        return f"{message}"
```

Comment:
In this case you could just do: `return message`, don't you?

Reply:
Done 4570dbed9b444b5084dbee6f758f3994cee3deb9